views/management/forms/instance-policy.hbs

```diff
@@ -0,0 +1,24 @@
+<div class="form-group">
+	<label>Titel</label>

```

Comment:
needs for attribute and linking to the proper input via id of that input

Reply:
done